src/test/java/org/folio/search/service/converter/ElasticsearchDocumentConverterTest.java

```diff
@@ -63,6 +63,8 @@ class ElasticsearchDocumentConverterTest {
   private SearchHits searchHits;
   @Mock
   private SearchHit searchHit;
+  @Mock
+  private SearchResponsePostProcessor<?> searchResponsePostProcessor;
```

Comment:
As I can see the mock is never used as well as map:
![image](https://github.com/folio-org/mod-search/assets/5904625/53d65d22-1c7f-4ba2-be3c-c94806b5cece)